src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -0,0 +1,92 @@
+"""Heberto Mayorquin, Authors: Saksham Sharda, Cody Baker, Ben Dichter."""
+from typing import Optional
+from pathlib import Path
+
+from pynwb.file import NWBFile
+
+from ....basedatainterface import BaseDataInterface
+from neuroconv.utils import dict_deep_update
+from neuroconv.utils import FilePathType, OptionalFilePathType
+from neuroconv.tools.nwb_helpers import make_or_load_nwbfile
+
+try:
+    from dlc2nwb.utils import auxiliaryfunctions, write_subject_to_nwb
+
+    HAVE_DLC2NWB = True
+except ImportError:
+    HAVE_DLC2NWB = False
+
+
+class DeepLabCutInterface(BaseDataInterface):
+    """Data interface for DeepLabCut datasets"""
+
+    def __init__(
+        self,
+        file_path: FilePathType,
+        config_file_path: FilePathType,
+        subject_name: str = "ind1",
+        verbose: bool = True,
+    ):
+        """
+        Interface for writing DLC's h5 files to nwb using dlc2nwb.
+        Parameters
+        ----------
+        file_path: FilePathType
+            path to the h5 file output by dlc.
+        config_file_path: FilePathType
+            path to .yml config file
+        subject_name: str
+            the name of the subject for which the nwbfile is to be created.
+        verbose: bool
+            controls verbosiy. True by default.
+        """
+        file_path = Path(file_path)
+        if "DLC" not in file_path.stem or ".h5" not in file_path.suffixes:
+            raise IOError("The file passed in is not a DeepLabCut h5 data file.")
+        assert HAVE_DLC2NWB, "to use this datainterface: 'pip install dlc2nwb'"
```

Comment:
@bendichter should the error message be for users to pip install 
`git+https://github.com/catalystneuro/DLC2NWB@add_to_existing_nwb` until this is merged to `dlc2nwb`?

Reply:
Would prefer to wait until it's merged upstream

Reply:
Done:
https://github.com/DeepLabCut/DLC2NWB/pull/10#event-7070987720

Reply:
That does not solve the problem here, right? doing pip install dlc2nwb is not going to install a version that contains this as it will revert to the latest release on PiPy.

Reply:
Yeah, I suppose now we're just waiting on it being re-released

Reply:
Release is done. I pinned the full requirements to the new version. Should be working now. I will resolve this conversation.